src/metrics.cpp

```diff
@@ -18,6 +18,30 @@ void Metrics::print(){
 }
 
 
+Metrics avg_metrics(std::vector<Metrics> vector_metrics){
+    Metrics metrics;
+    unsigned int sum_k = 0;
+    unsigned int sum_amplitude = 0;
+    double sum_selfCompression = 0;
+    double sum_normalizedCompression = 0;
+
+    for (auto a:vector_metrics){
+        sum_k+=a.k;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'sum_k' is assigned a value that is never used.](https://app.codacy.com/app/jorgeMFS/TMCompression/pullRequest?prid=4494377)